### PR TITLE
Css fixes for borders and dark theme colors

### PR DESCRIPTION
**Pull request by yebrahim**
Fixes https://github.com/googledatalab/datalab/issues/1226

Also fixes output text colors in dark theme.

### CODE REVIEW

File: sources/web/datalab/static/datalab.css [hunk 1]
```diff
@@ -13,7 +13,7 @@
  */
 
 /* Font Imports */
-@import url(https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Open+Sans:400,700|Lato:400,700&subset=latin,latin-ext);
+@import url(https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Open+Sans:400,700&subset=latin,latin-ext);
```
Comment (nikhilk) [inline sources/web/datalab/static/datalab.css]: Don't restrict Open Sans to just latin. We want to ensure non-latin characters don't show up as squares. Remove "|Lato:400,700&subset=latin,latin-ext"

Comment (yebrahim) [inline sources/web/datalab/static/datalab.css]: Done.

File: sources/web/datalab/static/light.css [hunk 2]
```diff
@@ -52,13 +57,18 @@ div.btn-toolbar .btn.active {
...
+  box-shadow: 0px 0px 6px 0px rgba(0, 0, 0, 0.3);
+}
 div.code_cell.session.completed div.input {
   border-left: solid 4px #337ab7;
 }
 div.code_cell div.input {
   background-color: #f7f7f7;
 }
-.CodeMirror-selected {
+.codemirror-selected {
```
Comment (nikhilk) [inline sources/web/datalab/static/light.css]: Just noticed this ... its .CodeMirror in other rules; just double checking this was intended (I believe these things are case-sensitive).

Comment (yebrahim) [inline sources/web/datalab/static/light.css]: Typo. Fixed.

**Pull request merged**